### PR TITLE
[NP-2037] Better arrow lines

### PR DESCRIPTION
**Pull request by KernelDeimos**
Arrow lines now never overlap capabilities, and lines will not cross in a way that makes relations ambiguous.

Screen Shot 2021-02-24 at 8 39 39 PM

### CODE REVIEW

File: src/foam/u2/svg/graph/ArrowDisplaceCellsPlacementPlan.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
```
Comment (kgrgreer) [inline src/foam/u2/svg/graph/ArrowDisplaceCellsPlacementPlan.js]: Is this a new file or just renamed?

File: src/foam/u2/svg/graph/ArrowDisplaceCellsPlacementPlan.js [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+      class: 'Map'
+    },
+    {
+      name: 'rowDisplacement_',
+      class: 'Map'
+    },
+    {
+      name: 'gridPlacementPlan',
+      class: 'FObjectProperty',
+      of: 'foam.u2.svg.map2d.GridPlacementPlan',
```
Comment (kgrgreer) [inline src/foam/u2/svg/graph/ArrowDisplaceCellsPlacementPlan.js]: no trailing commas

File: src/foam/u2/svg/graph/DAGView.js [hunk 3]
```diff
@@ -0,0 +1,337 @@
+foam.CLASS({
```
Comment (kgrgreer) [inline src/foam/u2/svg/graph/DAGView.js:7]: copyright

File: src/foam/u2/svg/map2d/PlacementPlan.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
```
Comment (kgrgreer) [inline src/foam/u2/svg/map2d/PlacementPlan.js]: year?

File: src/foam/u2/svg/graph/ArrowDisplaceCellsPlacementPlan.js (hunk 1 above)
Comment (KernelDeimos) [reply src/foam/u2/svg/graph/ArrowDisplaceCellsPlacementPlan.js]: yes this is new - updated the copyright header

**Pull request merged**